### PR TITLE
Design adjustments

### PR DESCRIPTION
**Pull request by motleydev**
**What**:
I've made a series of design changes such as layout, margins and max-width settings.

**Why**:
The readability is not very good in its current state.

**How**:
Exclusively through style adjustments.

**Before**

screen shot 2017-07-10 at 20 58 05

**After**

screen shot 2017-07-10 at 20 59 48

My feelings won't be hurt if you hate my style preference! :) Thanks!

### CODE REVIEW

**Comment by kentcdodds** [comment]
This is great! Thank you so much for these contributions! Do you think you could follow [the instructions](https://github.com/kentcdodds/glamorous-website/blob/master/CONTRIBUTING.md#making-a-pull-request) for deploying a version of this for people to look at?

I was just thinking yesterday that the font was too thin, I think you may have made it too thick though, maybe somewhere in the middle? :wink:

File: styles/base.js [hunk 1]
```diff
@@ -144,7 +141,7 @@ export default () => `
   /* to be used by our code blocks */
   .prism-code {
     display: block;
-    white-space: pre;
+    white-space: pre-wrap !important;
```
Comment (kentcdodds) [inline styles/base.js]: If `!important` is needed here, let's add a comment explaining why.

File: components/docs-page.js [hunk 2]
```diff
@@ -68,12 +71,12 @@ function PageSections({title, note, heading, sections}) {
       <PageWrapper>
         <Title dangerouslySetInnerHTML={{__html: mdToHTMLUnwrapped(title)}} />
         <Div
-          maxWidth="70rem"
+          maxWidth="35rem"
```
Comment (kentcdodds) [inline components/docs-page.js]: I feel that `35rem` is a little too narrow... How about `50rem`? I think that's where we had it at one time...

**Comment by motleydev** [comment]
Your wish is [my url link](https://en-glamorous-rnbcwlrzvp.now.sh/). 

* I'll wait for feedback after you've poked around the live version then I'll add the comment about stuff like the `important!`.
* Sadly I'm not sure there is a _lighter_ version of the font, that's already 400 and the next lower version in the family is 100.
* 50em still seems to break the [50-60 char/line](https://baymard.com/blog/line-length-readability) But I'm happy to oblige. :) Let me know what you think of it "live"

Do it Live

**Comment by kentcdodds** [comment]
Ok, the font actually looks great :+1: thanks!

I still think it's a bit too narrow. Especially on my 4k monitor 😅 

screen shot 2017-07-10 at 2 41 22 pm

In all seriousness, it also makes the interactive code blocks pretty narrow:

screen shot 2017-07-10 at 2 41 26 pm

We take a lot of hints from styled-components.com and they're doing `56rem`. I think that `50rem` should be ok without causing any trouble for folks.

Thanks again! This is great 👍

File: styles/base.js [hunk 3]
```diff
@@ -144,8 +141,10 @@ export default () => `
   /* to be used by our code blocks */
   .prism-code {
     display: block;
-    white-space: pre;
-
+    white-space: pre-wrap !important;
```
Comment (kentcdodds) [inline styles/base.js]: These styles in particular are borrowed from react-live, which injects them again. We could probably tell react-live to not inject the styles. But this is ok for now.

**Comment by kentcdodds** [comment]
Would you like to [add yourself](../blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) to the [contributors table](../#contributors)?

**Comment by motleydev** [comment]
NP! Happy to help! I've learned a lot from watching your Egghead videos so I'm stoked to be able to give a little back!

**Comment by kentcdodds** [comment]
Haha, @motleydev, could you fix this on [the advanced page](https://rc.glamorous.rocks/advanced/) please? 😂 

screen shot 2017-07-11 at 9 19 27 pm

**Comment by kentcdodds** [comment]
Those are shields.io badges and should _not_ have a width of 100% 😄

**Comment by motleydev** [comment]
Just say down to do this but looks like @tdeschryver just beat me to it with [#231](https://github.com/kentcdodds/glamorous-website/pull/231)!

**Comment by tdeschryver** [comment]
Oh sorry, I didn't see this :smile:

**Comment by kentcdodds** [comment]
That went pretty quick! You seem to be interested in design! Maybe you can help with this! https://github.com/kentcdodds/glamorous-website/pull/219

**Comment by motleydev** [comment]
Love to! Heading in to take a look.

**Comment by motleydev** [comment]
Wrapping up my day here in DE. I'm going to have to take a look at this tomorrow.

**Pull request merged**